Bottom: 1f9d8842f5755ce5d759fc64d75318d16d1172d8
Top:    58cbb60f51d474551a64162d8fcb6c01df074eb3
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-29 15:43:12 -0700

Refresh of rstream-opt

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 123f1a1..a863819 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -543,7 +543,6 @@ static int run(void)
 	       "name", "bytes", "xfers", "iters", "total", "time", "Gb/sec", "usec/xfer");
 	if (!custom) {
 		optimization = opt_latency;
-optimization = opt_bandwidth;
 		ret = dst_addr ? client_connect() : server_connect();
 		if (ret)
 			goto free;
@@ -552,13 +551,11 @@ optimization = opt_bandwidth;
 			if (test_size[i].option > size_option)
 				continue;
 			init_latency_test(test_size[i].size);
-init_bandwidth_test(test_size[i].size);
 			run_test();
 		}
 		rs_shutdown(rs, SHUT_RDWR);
 		rs_close(rs);
 
-sleep(1);
 		optimization = opt_bandwidth;
 		ret = dst_addr ? client_connect() : server_connect();
 		if (ret)
